t/unit/utils/test_platforms.py

```diff
@@ -689,6 +689,15 @@ def test_remove_if_stale_no_pidfile(self):
         assert p.remove_if_stale()
         p.remove.assert_called_with()
 
```

Comment:
is the change of file permission 100644 → 100755  needed here? or it was done by accident?

Reply:
Mistake corrected.